scenarios/setup.xml

```diff
@@ -70,10 +70,10 @@
                   exec="onos-check-apps ${OC#} org.omecproject.up4 includes"
                   requires="Up4-App-Install-Local-Build"/>
             <step name="Up4-Set-Log-Debug-${#}"
-                  exec="onos ${OC#} log:set DEBUG org.omecproject.up4"
+                  exec="ssh -q -p ${OP#} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${OC#} log:set DEBUG org.omecproject.up4"
                   requires="Onos-Check-Up4-App-${#}"/>
             <step name="Onos-Enable-Grpc-Log-${#}"
-                  exec="onos ${OC#} cfg set org.onosproject.grpc.ctl.GrpcChannelControllerImpl enableMessageLog true"
+                  exec="ssh -q -p ${OP#} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${OC#} cfg set org.onosproject.grpc.ctl.GrpcChannelControllerImpl enableMessageLog true"
```

Comment:
Can we move this to a different PR? Maybe adding a convenience script under `scenarios/bin` folder